bridge/evm/deploy_configs/31337.json

```diff
@@ -2,9 +2,9 @@
   "committeeMemberStake": [2500, 2500, 2500, 2500],
   "committeeMembers": ["0x68b43fd906c0b8f024a18c56e06744f7c6157c65", "0xacaef39832cb995c4e049437a3e2ec6a7bad1ab5", "0x8061f127910e8ef56f16a2c411220bad25d61444", "0x508f3f1ff45f4ca3d8e86cdcc91445f00acc59fc"],
   "minCommitteeStakeRequired": 10000,
-  "sourceChainId": 0,
+  "sourceChainId": 12,
```

Comment:
12 is EthCustom, just want to confirm this is the correct chain id for this config?